build/compile-protbuf.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Should we include this in Travis' pipeline?

Reply:
should/do we compile then on demand like in the java lib?